build-ps/percona-server.spec

```diff
@@ -986,6 +986,12 @@ if [ $? -eq 1 -a "$SERVER_TO_START" = "true" ] ; then
 fi
 %endif
 
+if [ ! -d %{_datadir}/mysql ]; then
+    pushd %{_datadir}
+    ln -s percona-server mysql
+    popd
+fi
```

Comment:
please add condition on preun to remove link if we are removing package.

Reply:
done